lib/engine/game/g_1866/stock_market.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require_relative '../../stock_market'
```

Comment:
is this class necessary? i try to avoid having too many of these base classes, but if it's the best way we can keep it

Reply:
I think it is necessary, i override the move_down and move_up methods. The stock market in 1866 is very special, i dont see any other game have any usefulness for that code. Atleast the move down code, its very specific. We can solve it with a bunch of constants or base-methods calls, but i think that would clutter down the base code more then just to keep the overridden class.